Automattic/jetpack
# Update Notice component in My Jetpack

pbNhbs-ag3-p2#comment-21411

## Proposed changes:

* This PR updates the Notice component used in My Jetpack and contains a little refactor on how setNotice works (to be more in line to the new component)

NOTE: this component does not apply the changes mentioned in the thread linked above, it just updates the component. The other changes will be done in a different PR

<img width="1701" alt="image" src="https://github.com/Automattic/jetpack/assets/5714212/3314d242-f7eb-4ed6-bdde-7e7debce305c">


### Other information:


## Jetpack product discussion


pbNhbs-ag3-p2#comment-21411

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Spin up a JN site with Jetpack Beta pointing to this feature branch
* Without connecting Jetpack, visit `/wp-admin/admin.php?page=my-jetpack`
* Dismiss the welcome banner and refresh the page
* Confirm that the Notice looks like the screenshot above
